### PR TITLE
Adding in the four modified files for posts

### PR DESCRIPTION
**Pull request by rogeryanda**

### CODE REVIEW

File: _posts/0000-01-34-rogeryanda.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+A fourth caption
```
Comment (rtsinclair) [inline _posts/0000-01-34-rogeryanda.md:6]: Nice job Roger

Comment (rtsinclair) [reply _posts/0000-01-34-rogeryanda.md:6]: What were you thinking!

**Review by rtsinclair — CHANGES_REQUESTED** [review]
I like your flow and coordination.

**Review by rtsinclair — APPROVED** [review]

**Comment by rogeryanda** [comment]
Testing the merge

**Pull request closed without merging**